### DISCUSSION TITLE
Repos without License

### DISCUSSION THREAD

**Discussion by JohannesPaulHoffmann** [community/community#75969] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,

there are Repos which come without a License (document), the software is then (mostly at least) intellectual property, or do I misunderstand?

Regards from Germany,
Johannes

**Comment by mmqeth** [comment]
You can refer to the discussion in the following link.
https://opensource.stackexchange.com/questions/1720/what-can-i-assume-if-a-publicly-published-project-has-no-license

**Comment by EliasAfara** [comment]
#ad

Check out [**Contribution Streak Maintainer**](https://www.npmjs.com/package/contribution-streak-maintainer), your go-to solution for effortlessly maintaining your GitHub contribution streak. 🚀

[![npm](https://img.shields.io/npm/v/contribution-streak-maintainer)](https://www.npmjs.com/package/contribution-streak-maintainer) [![npm](https://img.shields.io/npm/dm/contribution-streak-maintainer)](https://www.npmjs.com/package/contribution-streak-maintainer) [![GitHub License](https://img.shields.io/github/license/devful/contribution-streak-maintainer)](https://github.com/devful/contribution-streak-maintainer/blob/main/LICENSE) [![GitHub Repo stars](https://img.shields.io/github/stars/devful/contribution-streak-maintainer)](https://github.com/devful/contribution-streak-maintainer/stargazers)

Do you struggle to keep up with your GitHub contribution streak? Is making daily manual contributions challenging? Say goodbye to those worries!

**Contribution Streak Maintainer** is a powerful npm package designed for seamless integration into your GitHub Actions workflow. By automating the generation and pushing of commits to your repository, this tool ensures your GitHub activity remains active, even on days when you're unable to make manual contributions.

### Key Features

- 📅 Automatic GitHub contribution streak maintenance
- 🛠️ Easy integration into GitHub Actions workflows
- 🌐 Supports a variety of workflows with customizable conditions
- 🤖 Effortlessly keeps your streak alive, allowing you to focus on your work

**Comment by EliasAfara** [comment]
When a software project does not include a license document, it means that the default copyright laws apply. In many jurisdictions, this would indeed imply that the software is considered the intellectual property of the creator or the organization that developed it. Without a license, others may not have the legal right to use, modify, distribute, or contribute to the code.

It's crucial to note that the absence of a license can lead to ambiguity regarding the permissions granted to others

**Comment by JohannesPaulHoffmann** [comment]
Well, Licenses are for grantin' rights, while legal laws (at least here in Germany) would fully let the rights (distribute, modiy (work with it) and so on) stay to the Owner/Author.

As GitHub put as default ~"all rights reserver", at least implied, it follows legal laws (at least in My Country).

Was really wonderin', but sure, if it applies the other way around(?).

OSS (You can see it), FOSS, You have more than OSS rights (mostly.

Thanks for the responses!

**EDIT**
